atc/atccmd/command_test.go

```diff
@@ -27,6 +27,20 @@ func (s *CommandSuite) TestLetsEncryptDefaultIsUpToDate() {
 	s.Equal(opt.Default, []string{autocert.DefaultACMEDirectory})
 }
 
+func (s *CommandSuite) TestInvalidConcurrencyLimit() {
```

Comment:
it probably then makes more sense for this test to assert on either the "no duplicates" rule.

Reply:
... since it's really an integration test asserting on the wiring (most importantly that the `Validate()` method is being called on the concurrency policy.